packages/suite/src/utils/wallet/__fixtures__/coinControlUtils.ts

```diff
@@ -0,0 +1,60 @@
+import * as coinControlUtils from '../coinControlUtils';
+
+const baseUtxo = {
+  txid: '1',
+  vout: 1,
+  amount: '100',
+  blockHeight: 100,
+  path: 'string',
+  confirmations: 100,
+};
+
+export const filterAndCategorizeUtxosParams: Array<{
```

Comment:
nit: I would prefer to have it as separate type and then do just `SomeCoolName[]`